webapp/channels/src/components/advanced_text_editor/formatting_bar/hooks.tsx

```diff
@@ -18,23 +18,24 @@ export function useGetLatest<T>(val: T) {
 
 const useResponsiveFormattingBar = (ref: React.RefObject<HTMLDivElement>): WideMode => {
     const [wideMode, setWideMode] = useState<WideMode>('wide');
-    const handleResize = debounce(() => {
+    const handleResize = useCallback(debounce(() => {
         if (ref.current?.clientWidth === undefined) {
             return;
         }
-
         if (ref.current.clientWidth > 640) {
             setWideMode('wide');
         }
-
         if (ref.current.clientWidth >= 424 && ref.current.clientWidth <= 640) {
             setWideMode('normal');
         }
-
         if (ref.current.clientWidth < 424) {
             setWideMode('narrow');
         }
-    }, 10);
+
+        if (ref.current.clientWidth < 310) {
+            setWideMode('min');
+        }
+    }, 10), [ref.current?.clientWidth]);
```

Comment:
[ref's should not](https://epicreact.dev/why-you-shouldnt-put-refs-in-a-dependency-array/) be used as dependency

Reply:
Thanks for the link. I fixed it.